### PR TITLE
feat: add sticky filter controls on organizations page

### LINKED ISSUE #1107: [Feature Request] Add sticky category/sort controls on Organizations page during scroll
##Feature Description

Add sticky category and sorting controls on the Organizations page to improve usability during long scrolling sessions.

## Problem It Solves

Currently, when users scroll through a long list of organizations, the category, complexity, and sorting controls move out of view. Users need to scroll back to the top repeatedly to modify sorting or filtering options, which affects navigation efficiency and overall user experience.

##Proposed Solution

Implement sticky positioning for the category, complexity, and sorting controls so they remain accessible while users scroll through the Organizations page.

The implementation would:

* Keep controls visible during scrolling
* Maintain responsiveness across desktop/tablet/mobile devices
* Preserve the current UI consistency and layout

## Alternatives Considered

* Floating filter button
* Full sticky filter panel
* Mobile-only filter drawer

However, making only the category/sort controls sticky provides a simpler and cleaner UX improvement without significantly changing the existing design.

## Mockups / Examples

A sticky controls section similar to modern e-commerce and dashboard interfaces where sorting/filter controls remain visible while browsing long lists.

##  Acceptance Criteria

* [ ] Category, complexity, and sort controls remain visible during scrolling
* [ ] Sticky behavior works properly across different screen sizes
* [ ] Existing UI alignment and spacing remain unaffected
* [ ] Smooth scrolling experience without layout glitches

## Additional Context

This enhancement would improve navigation usability and make browsing organizations more efficient, especially for users exploring large organization lists.

##Reproduction / Current Behavior

Current Behavior

When users scroll through the Organizations page, the category, complexity, and sorting controls disappear from view. Users must scroll back to the top repeatedly to modify filtering or sorting options.

Expected Behavior

The category, complexity, and sorting controls should remain visible while scrolling through the organization list for easier navigation and filtering.

##Steps to Reproduce

1. Open the Organizations page
2. Scroll down through the organization cards
3. Observe that the filter and sorting controls are no longer visible
4. Users must manually scroll back to the top to access them again

## Contributor Checklist

 I am participating via GSSoC
 I have read the contribution guidelines
 I checked for existing issues before creating this

I would like to work on this issue under GSSoC. Kindly assign this issue to me.

### PR DESCRIPTION
**Pull request by Susant-Raj03**
## Program
gssoc26

## Description
Added sticky filter controls for category, complexity, and sorting on the Organizations page so they remain visible while scrolling.

## Related Issue
Closes #1107

## Type of Change
- [x] Enhancement
- [x] UI Improvement

## Checklist
- [x] Code follows project style guidelines
- [x] Tested locally
- [x] No existing functionality broken
- [x] Responsive design maintained

## Changes Made
- Added sticky behavior to filter controls
- Improved scrolling usability
- Maintained responsive layout
- Added subtle UI polish for better UX

Kindly review this PR and let me know if any changes are required.

### CODE REVIEW

**Review by Anushreebasics — DISMISSED** [review]
DCO sign-off and program classification missing

**Review by nitinog10 — DISMISSED** [review]
Nice enhancement! The sticky filter controls improve usability on long organization lists, and the responsive behavior looks well handled. The implementation is clean and scoped properly.LGTM!

**Review by Anushreebasics — DISMISSED** [review]
The program classification CI is still failing otherwise, LGTM!

**Comment by Anushreebasics** [comment]
@Susant-Raj03 pls resolve conflicts

**Comment by Susant-Raj03** [comment]
> @Susant-Raj03 pls resolve conflicts

@Anushreebasics Sir,please check it again.

**Review by Anushreebasics — APPROVED** [review]
LGTM!

File: index.html [hunk 1]
```diff
@@ -1037,50 +1037,7 @@ <h3 class="font-headline font-extrabold text-lg sm:text-xl">2026 Timeline</h3>
...
-        <input id="matchAllLanguagesToggle" type="checkbox" class="rounded border-zinc-300 text-primary focus:ring-primary" aria-describedby="languageLogicHint"/>
-        <span class="text-xs text-zinc-600 font-medium">Match all selected languages </span>
-      </label>
-      <span id="languageLogicHint" 
-        class="text-xs text-primary font-medium italic">
-      </span>
-    </div>
-  </div>
-
-  <div class="flex flex-wrap items-center justify-between gap-4 mb-8">
```
Comment (S3DFX-CYBER) [inline index.html:1093]: Why have u removed this part ?

**Review by S3DFX-CYBER — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Susant-Raj03) [reply index.html:1093]: > Why have u removed this part ?

sorry sir ,I did not intend to remove this functionality. Let me check the conflict resolution and restore the removed section if it was accidentally deleted.

**Comment by rajesh-puripanda** [comment]
I would like to add sticky filter controls on the organizations page for GSSoC 2026!

**Plan:**
1. Identify the filter container elements (language pills, sort dropdowns, search)
2. Apply position: sticky with proper top offset and z-index so filters stay visible while scrolling
3. Add a subtle backdrop blur or background to maintain readability over page content
4. Ensure the sticky behavior works on mobile (compact sticky bar)
5. Handle edge cases: when filter bar is already at top on load, do not show duplicate
6. Test across all pages that have filter controls

This is a clean UX improvement I can implement efficiently. Please assign!

**Review by Anushreebasics — APPROVED** [review]
pls squash commits to just one
